ui/analyse/src/ctrl.ts

```diff
@@ -574,7 +575,8 @@ export default class AnalyseCtrl {
   }
 
   onPremoveSet = () => {
-    if (this.study) this.study.onPremoveSet();
+    if (this.node.dests === '') alert('Too many moves for a lichess board.');
```

Comment:
That's a weird assumption to make and one that would probably backfire. Can we use the ply number instead?